src/main/kotlin/com/terraformation/backend/api/Extensions.kt

```diff
@@ -2,7 +2,10 @@ package com.terraformation.backend.api
 
 import com.terraformation.backend.file.SizedInputStream
 import jakarta.ws.rs.NotSupportedException
+import java.io.ByteArrayOutputStream
 import org.apache.tika.mime.MimeTypes
+import org.geotools.api.feature.simple.SimpleFeature
+import org.geotools.geojson.feature.FeatureJSON
```

Comment:
This is a handy tool to output to GeoJSON. It doesn't seem to support using a custom serializer. At some point we can build our own GeoJSON builder.